examples/gameroom/gameroom-app/src/store/modules/nodes.ts

```diff
@@ -30,9 +30,10 @@ const getters = {
 };
 
 const actions = {
-  async listNodes({ commit }: any) {
-    const nodes = await listNodes();
-    commit('setNodes', nodes);
+  async listNodesWithoutLocal({ commit }: any, localBrand: string) {
```

Comment:
Was this change supposed to be included in this commit?

Reply:
Oh no, it wasn't. Think I accidentally scooped up some unrelated changes when I pushed to update the commit message. Fixed now!